src/components/fields/SchemaField.js

```diff
@@ -91,7 +91,7 @@ function DefaultTemplate(props) {
   }
 
   return (
-    <div className={classNames}>
+    <div id={id} className={classNames}>
```

Comment:
That would duplicate the id, while they should be unique to the document; we could suffix the target wrapping component id.